app/controllers/items_controller.rb

```diff
@@ -0,0 +1,60 @@
+class ItemsController < ApplicationController
+  before_action :set_item, only: [:show, :edit, :update, :destroy]
+  def index
+    if params[:merchant_id]
+      @merchant = Merchant.find(params[:merchant_id])
+      @items = @merchant.items
+    else
+      @items = Item.all
+    end
+    @items = sort_items(@items, params[:sort]) if params[:sort]
+  end
+
+  def sort_items(items, method)
+    if method == 'a-z'
+      items.sort_by { |item| item.name }
+    elsif method == 'z-a'
+      items.sort_by { |item| item.name }.reverse
+    elsif method == 'min-price'
+      items.sort_by { |item| item.price }
+    elsif method == 'max-price'
+      items.sort_by { |item| item.price }.reverse
+    end
+  end
```

Comment:
You're right in your comment that you left on this PR. You could/should be using activerecord for these queries. Using the method you have write now, you can sort using the order method. For example, to sort your items from highest to lowest by price, you can do something like this: 
```ruby
items.order(price: :desc)
```
or if you wanted to order lowest to highest 
```ruby
items.order(price: :asc)
```
The default to order is alphabetically if it's strings that you want to order. So, if you wanted to order alphabetically, just select the attribute you want to order by and the :desc and the :asc determines if it's a-z or z-a. For example, 

```ruby 
items.order(name: :asc)
```